src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -5789,14 +5812,14 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
     constexpr int MaxZoneEquipmentIdx{MaxZoneEquipmentOn - 1};
     std::array<int, MaxZoneEquipmentOn> HeatingPriorityStack{};
     std::array<int, MaxZoneEquipmentOn> CoolingPriorityStack{};
-    std::array<ConvectionConstants::InConvFlowRegime, MaxZoneEquipmentOn> FlowRegimeStack{};
-    FlowRegimeStack.fill(ConvectionConstants::InConvFlowRegime::Invalid);
+    std::array<InConvFlowRegime, MaxZoneEquipmentOn> FlowRegimeStack{};
```

Comment:
These lines just changed because now the enum is in the top of this cc file, not in a different namespace.